README.md

```diff
@@ -1,37 +1,48 @@
-## Midweight Developer Code Test
+## Comic Relief Engineer Code Test
 
-The aim of the code test is to create an API endpoint that an API user can submit a post code to, which
-will reutrn a list of potential addresses which have been enhanced with latitudes and longitudes.
+The aim of the code test is to create an API endpoint that an API user can submit a postcode to, which
+will return a list of potential addresses which have been **enhanced with latitudes and longitudes**.
 
-You can find an example of how the end result should function below,
+You can find an example of how the end result might function below:
 
-```
-https://s4aily6l6c.execute-api.eu-west-2.amazonaws.com/production/postcode/SE17TP
-```
+https://poqr2v25cb.execute-api.us-east-1.amazonaws.com/prod/postcode/SE17TP
 
-As part of your code, You will need to send a GET request to our post code api to get the list of addresses,
+You may add your own formatting, styles etc to the output, change error handling, etc.
 
-```
-https://lookups.sls.comicrelief.com/postcode/lookup/SE17TP
-```
+As part of your code, You will need to send a GET request to our post code api to get the list of addresses:
 
-You will then need to send a GET request to the postcode.io api to get latitudes and longitudes,
+[https://lookups.sls.comicrelief.com/postcode/lookup/SE17TP](https://lookups.sls.comicrelief.com/postcode/lookup/SE17TP)
 
-```
-https://api.postcodes.io/postcodes/SE17TP
-```
+You will then need to send a GET request to the postcode.io api to get latitudes and longitudes:
+
+[https://api.postcodes.io/postcodes/SE17TP](https://api.postcodes.io/postcodes/SE17TP)
 
 You will then need to join the latitude and longitude data from postcode.io onto the data from our
 postcode api and return the data to the user.
 
-### Local Developemnt
+### Local Development
 
-To run the endpoint locally you can run the following command in the command line,
-```
-sls offline start
+Prerequisites:
+- Serverless installed globally in order to run the function locally using serverless offline
+- NodeJS
+- Yarn
+
+Clone the repo and then execute
+
+```bash
+yarn build
```

Comment:
Could still use NPM here; doesn't seem to be any other Yarn commands in the repo

Reply:
I thought because there's a `yarn.lock` file it was tied to yarn rather than NPM for package management?

Reply:
Mmmm I don't think that'd stop anyone using npm; the app just wouldn't be tied to specific versions, at least nothing beyond what it specified in the package.json.

Anyways, it's irrelevant really; was just wondering if it's requiring a lot of stuff to be installed globally but it's not like it's a big deal. Fahgedddddditabout. Sometimes PR reviews just become "I need to say something thoughtful" exercises